tools/load-examples.groovy

```diff
@@ -0,0 +1,257 @@
+		}
+
+		// Unzip the examples
+		def examples_zip = new File(examples_zip_path)
+		if (!examples_zip.exists())
+		{
+			println "[" + scriptFileName + "] " + examples_zip_path + " not found!"
+			return
+
+		}
+		unzipFile(examples_zip, example_dir_path)
+		println "[" + scriptFileName + "] " + examples_zip_path + " extracted!"
+
+		def target_dir_path = ""
+		def bucket = ""
+
+		// Start by finding the next template dir index
+		def templates_dirs_list = []
+		new File(workflow_templates_dir_path).eachDir { dir ->
+			templates_dirs_list << dir.getName().toInteger()
+		}
+		def template_dir_name = "1"
+		if (!templates_dirs_list.isEmpty())
+			template_dir_name = (templates_dirs_list.sort().last() + 1) + ""
+		println "[" + scriptFileName + "] Next template dir name " + template_dir_name
+
+		// For POST queries
+		Main.class.getClass().getResource(new File("../dist/lib/httpclient-4.5.2.jar").absolutePath);
+		def boundary = "---------------" + UUID.randomUUID().toString();
+
+		// For each example directory
+		example_dir.eachDir() { dir ->
+
+			def metadata_file = new File(dir.absolutePath, "METADATA.json")
+			if (metadata_file.exists())
+			{
+				println "[" + scriptFileName + "] Parsing " + metadata_file.absolutePath
+
+				// From json to map
+				def slurper = new groovy.json.JsonSlurper()
+				def metadata_file_map = (Map) slurper.parseText(metadata_file.text)
+
+				def catalog_map = metadata_file_map.get("catalog")
+
+
+				// DATASPACE SECTION /////////////////////////////
+
+
+				def dataspace_map = null
+				if ((dataspace_map = catalog_map.get("dataspace")) != null)
+				{
+					// Retrieve the targeted directory path
+					def target = dataspace_map.get("target")
+					if(target == "global")
+						target_dir_path = global_space_path
+
+					// Copy all files into the targeted directory
+					dataspace_map.get("files").each { file_relative_path ->
+						def file_src = new File(dir.absolutePath, file_relative_path)
+						def file_src_path = file_src.absolutePath
+						def file_name = file_src.getName()
+						def file_dest = new File(target_dir_path, file_name)
+						def file_dest_path = file_dest.absolutePath
+						Files.copy(Paths.get(file_src_path), Paths.get(file_dest_path), StandardCopyOption.REPLACE_EXISTING)
+					}
+				}
+
+
+				// BUCKET SECTION /////////////////////////////
+
+
+				bucket = catalog_map.get("bucket")
+
+				// Does the bucket already exist? -------------
+				// GET QUERY
+				def list_buckets_rest_query = workflow_catalog_url + "/buckets?owner=" + bucket_owner
+				def response =  new URL(list_buckets_rest_query).text
+				def object_list = slurper.parseText(response.toString())
+				def object_found = object_list.find {object -> object.name == bucket}
+
+				println "[" + scriptFileName + "] bucket " + bucket + " found? " + (object_found != null)
+
+				// Create a bucket if needed -------------
+				def bucket_id = null
+				if (object_found)
+					bucket_id = object_found.id
+				else {
```

Comment:
It is considered as good practice to always put brace after a if. Maybe for single line if it is ok to not put them but here there is a else after. Could you add brace after the if ?